source/fuzz/transformation_inline_function.h

```diff
@@ -67,8 +67,9 @@ class TransformationInlineFunction : public Transformation {
 
   // Inline |instruction_to_be_inlined| by setting its ids to the corresponding
   // ids in |result_id_map|.
-  void AdaptInlinedInstruction(opt::IRContext* ir_context,
-                               opt::Instruction* instruction) const;
+  void AdaptInlinedInstruction(
```

Comment:
I would probably be good to mention in the docs for `IsApplicable` that we are using overflow ids.

Reply:
Done.